### ISSUE TITLE
shim HTMLElement in can-vdom

### ISSUE THREAD

**Issue by frank-dspeed** [canjs/canjs#5439]
As CanJS wants to move forward to WebComponents we need to Shim HTMLElement Complet to allow SSR for can-stache-element the current workaround is to use can-component

**Comment by matthewp** [comment]
Yep, this does need to happen. Probably better as an issue in can-vdom though.

**Comment by phillipskevin** [comment]
This is in progress in https://github.com/canjs/can-vdom/pull/70.
